tests/Runtimes/RuntimesTest.php

```diff
@@ -116,7 +126,7 @@ public function testSupportedRuntimes()
     {
         $this->assertNotEmpty($this->instance->get('node'));
         $this->assertNotEmpty($this->instance->getAll());
-        $this->assertCount(15, $this->instance->getAll(supported: false));
+        $this->assertCount(16, $this->instance->getAll(supported: false));
```

Comment:
@TorstenDittmann don't know what's happening here, but as I added new runtime, I incremented by 1 to pass the test :)

Reply:
```suggestion
        $this->assertNotEmpty($this->instance->getAll(supported: false));
```

I think we should do this - so it is easier for us in the future 👍🏻